packages/sdks/src/functions/track/index.ts

```diff
@@ -118,7 +123,9 @@ export async function _track(eventProps: EventProps) {
     return;
   }
 
-  return fetch(`https://cdn.builder.io/api/v1/track`, {
+  const baseUrl = eventProps.apiHost || 'https://cdn.builder.io';
```

Comment:
You have to make sure that `apiHost` isn't included in the call to `createEvent(eventProps)`, or we will be sending the `apiHost` property with every tracking event.